src/frontend/typer/søknad.ts

```diff
@@ -31,6 +32,7 @@ export interface OppholdUtenforNorge {
 export interface Aktivitet {
     utdanning: EnumFelt<JaNei>;
     aktivitet: EnumFlereValgFelt<string>;
+    annenAktivitet: EnumFelt<AnnenAktivitetType>;
```

Comment:
annenAktivitet er ikke påkrevd
```suggestion
    annenAktivitet: EnumFelt<AnnenAktivitetType> | undefined;
```

Reply:
Fixed: 366b29a74e477a8e270265fe16d20220b54858b4